roles/container_runtime/templates/registries.conf

```diff
@@ -13,9 +13,9 @@
 #registries:
 #  - registry.redhat.io
 
-{% if l2_docker_additional_registries %}
+{% if l_additional_crio_registries %}
```

Comment:
There is no need to change anything in this file.

Reply:
Right, `l_insecure_crio_registries` just uses `l2_docker_insecure_registries` - I saw that but wasn't sure. Thanks! Fixed.